test/test_mq.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(not(target_env = "ohos"))]
```

Comment:
Please do it here: https://github.com/nix-rust/nix/blob/299febac5aceda2f41825f8daa95ce469c0f74e3/test/test.rs#L16-L17